### ISSUE TITLE
Some tiled windows require to be "unmaximised" to show borders. 

### ISSUE THREAD

**Issue by abrac** [esjeon/krohnkite#116]
**Symptom**
Some applications open without a border, and I have to press the "unmaximise" button to make the border appear. It doesn't happen with all applications. Sometimes borders show for a certain application, but on another day, they don't.

Example gif:
Notice how the borders don't appear when I initially open the terminals. Only when I press the unmaximise button, do the borders appear. 
![borders](https://user-images.githubusercontent.com/55248999/94993491-ba5d3700-0591-11eb-986c-1d6c7e2121a7.gif)

**How to Reproduce**
1. Install Krohnkite
2. Apply "hide window titles" and "allow resizing maximsed windows" from _Window Decorations_ menu in _Settings_.
3. Change border colors as described in the [readme](https://github.com/esjeon/krohnkite#changing-border-colors)
4. Open new windows
5. Some will appear without borders

**Expected behavior**
Window should appear with borders by default.

**Environment**
 - Distro: Manjaro
 - KWin version: . 5.19.5-1
 - Krohnkite version: 0.7
 - List of KWin scripts in use: None besides Krohnkite
 - Number of displays: 1 
 - Resolution: 1366x768 

**Notes**
Could this be something to do with the way KWin "remembers" for each application whether windows should be maximised by default?

**Comment by alxlg** [comment]
Do you mean what's discussed in #64? If so the only way to completely fix this is having KWin implement an API that tells if a window is maximized or not.

**Comment by abrac** [comment]
Hi @alxlg. No, it's not the same issue. Perhaps I worded the title of my issue badly. Basically, some of my windows appear without borders in their **tiled** state. But for some reason, pressing the unmaximise button makes them show their borders. If I press the maximise button a third time, the window maximises for real.

I will try using the latest git version of Krohnkite. Perhaps new commits have fixed my issue...

**Comment by esjeon** [comment]
Hi, @abrac.

You'll want to enable "Allow resizing maximized windows from window edges" in Breeze Window Decoration settings. This will force render borders of the maximized windows, which should solve the problem for the most usecases.

![Screenshot_20201005_011459](https://user-images.githubusercontent.com/2756834/95020797-4a8da000-06a8-11eb-9212-61cbddc93b6a.png)

**Comment by abrac** [comment]
Hi @esjeon. Thanks for your comment! I already had the option ticked -- but your comment still made me realise the root of the problem. I realised that maximised windows had their borders hidden, even when Krohnkite was disabled. I found that my "Window Buttons" plasma applet had a setting which could override the option. 

See screenshot below:
![image](https://user-images.githubusercontent.com/55248999/95024412-0e941400-0683-11eb-8c08-0cbcbed6d127.png)
